notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>Please change all references of Vertex SDK to use the official name of Vertex AI SDK for Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I have changed the name to be 'Vertex aI SDK for Python'</p>